cni/network/invoker_cns.go

```diff
@@ -81,7 +81,7 @@ func (invoker *CNSIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, erro
 	response, err := invoker.cnsClient.RequestIPAddress(context.TODO(), ipconfig)
 	if err != nil {
 		log.Printf("Failed to get IP address from CNS with error %v, response: %v", err, response)
-		return IPAMAddResult{}, errors.Wrap(err, "Failed to get IP address from CNS with error: %w")
+		return IPAMAddResult{}, errors.Wrap(err, "Failed to get IP address from CNS with error")
```

Comment:
```suggestion
		return IPAMAddResult{}, errors.Wrap(err, "failed to get IP address from CNS")
```
since you're fixing this, error strings should also not be sentence case (and we can drop the "with error" because of course it is)